files-community/Files
# Fix: File extensions not showing up at Recent Files

**Resolved / Related Issues**
   Closes #15108

**Validation**
How did you test these changes?
   - [X] Was this change approved? Yes, at #15108
   1. Check Recent Files
   2. Click settings button 
   3. Folder/Files -> Toggle Show File Extensions
   4. Check Recent Files again

**Screenshots**
![image](https://github.com/files-community/Files/assets/52143624/b0d8636d-91dc-4c3d-9a8d-f6d1289b65b1)
![image](https://github.com/files-community/Files/assets/52143624/2cd278e0-49e3-4cd7-84b4-68094a53b73e)
